app/controllers/admin/categories_controller.rb

```diff
@@ -0,0 +1,45 @@
+class Admin::CategoriesController < Admin::BaseController
+  load_permissions_and_authorize_resource find_by: :slug
+
+  def index
+    @category_grid = initialize_grid(@categories, include: :categorizations,
+                                                  order: :title,
+                                                  order_direction: :asc)
```

Comment:
Asc is the default ordering in SQL, but I suppose it doesn't hurt to write it out.